tools/get-version.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Retrieving the version from jetpack.php file
+PHP_VERSION=`cat jetpack.php | grep '* Version' | cut -d ':' -f2`
```

Comment:
Should we limit this so just a few lines in the header? so head -15 instead of cat? 
This way if some some reason some one adds `* Version:` to the change the build script?